requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django==4.1
+Django==4.2.6
```

Comment:
https://docs.djangoproject.com/en/5.0/releases/4.2.9/

safety.io says there are CVEs in 4.2.6. Bump to latest subminor.

Reply:
Change